README.md

```diff
@@ -56,16 +47,28 @@ _Note_: Feel free to use the `nightly` tag to test the latest version, i.e. `git
 ### Linux
 
 - [Download the latest release](https://github.com/gitpod-io/openvscode-server/releases/latest)
-- Untar and run the server:
-```bash
-tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
-cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
-./server.sh
-```
-- Visit the URL printed in your terminal. Just going to port 3000 won't work, because VS Code requires that you provide a uniquely generated security token to prevent unauthorized access.
+- Untar and run the server
+	```bash
+	tar -xzf openvscode-server-v${OPENVSCODE_SERVER_VERSION}.tar.gz
+	cd openvscode-server-v${OPENVSCODE_SERVER_VERSION}
+	./bin/openvscode-server # you can add arguments here, use --help to list all of the possible options
+	```
+
+  From the possible entrypoint arguments, the most notable ones are
+	- `--port` - the port number to start the server on, this is 3000 by default
```

Comment:
We should mention `--without-connection-token` too and point out that this is the default in the docker image
```suggestion
	- `--port` - the port number to start the server on, this is 3000 by default
	- `--without-connection-token` - used by default in the docker image
```